src/components/Auth/AuthModal/AuthModal.jsx

```diff
@@ -0,0 +1,205 @@
+import {
+  CircularProgress,
+  Grid,
+  makeStyles,
+  Tab,
+  Tabs,
+} from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import React from "react";
+
+import ConfirmEmailPaper from "../AuthForms/ConfirmEmailForm";
+import LoginForm from "../AuthForms/LoginForm";
+import ResetPasswordForm from "../AuthForms/ResetPasswordForm";
+import SignupForm from "../AuthForms/SignupForm";
+
+import "./AuthModal.css";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    "& > *": {
+      margin: theme.spacing(1),
+      width: 300,
+    },
+  },
+  gridContainer: {
+    padding: theme.spacing(2),
+  },
+}));
+const AuthDialog = props => {
```

Comment:
```suggestion

const AuthDialog = props => {
```

Newlines between components (+readability)